### PR TITLE
Added logTeleconferenceDeviceQuality function under Call collection

### PR DESCRIPTION
**Pull request by dongkyunn**
Added the function for 3rd party CVI partners to report the media quality data to us.

Cloud Video Interop (CVI) bot represents Video teleconferencing (VTC) devices and acts as a back-to-back agent for VTC device in a conference call.  Since a CVI bot is in the middle of VTC and Teams infrastructure as a VTC proxy, it has two media legs. One media leg is between CVI bot and Teams infrastructure such as MC/MP or Teams client and the other media leg is between CVI bot and VTC device. 
The third party partners own the VTC media leg and Teams infrastructure cannot access the quality data of the third party call leg.  This API allows the CVI partners to provide their quality data.

### CODE REVIEW

File: api-reference/beta/resources/teleconferencedeviceaudioquality.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents video teleconferencing device audio quality data.
+
+## Properties
+
+**teleconferenceDeviceAudioQuality** inherits all the properties of the [teleconferenceDeviceMediaQuality](teleconferencedevicemediaquality.md) resource.
+
```
Comment (Lauragra) [inline api-reference/beta/resources/teleconferencedeviceaudioquality.md]: For this topic, please add a ## JSON representation section that includes the properties of the parent resource. This is required for API doctor validation.

File: api-reference/beta/resources/teleconferencedevicescreensharingquality.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents video teleconferencing device screen-sharing quality data.
+
+## Properties
+
+The **teleconferenceDeviceScreenSharingQuality** inherits all the properties from [teleconferenceDeviceVideoQuality](teleconferencedevicevideoquality.md).
+
```
Comment (Lauragra) [inline api-reference/beta/resources/teleconferencedevicescreensharingquality.md]: Add the ## JSON representation section here too with the parent resource properties.

File: api-reference/beta/resources/teleconferencedeviceaudioquality.md (hunk 1 above)
Comment (dongkyunn) [reply api-reference/beta/resources/teleconferencedeviceaudioquality.md]: Will update. Thanks!

File: api-reference/beta/resources/teleconferencedevicescreensharingquality.md (hunk 2 above)
Comment (dongkyunn) [reply api-reference/beta/resources/teleconferencedevicescreensharingquality.md]: Will add. Thanks!

**Pull request merged**